srcpkgs/libmypaint/template

```diff
@@ -23,9 +24,8 @@ libmypaint-devel_package() {
 		vmove usr/include
 		vmove usr/lib/pkgconfig
 		vmove "usr/lib/*.so"
+		if [ -z "$CROSS_BUILD" ]; then
+			vmove "usr/share/gir-*"
+		fi
```

Comment:
Ok, these are *new* files. At least they don't seem to exist in the current version of the package.

In that case, it should be made a build option and `build_options_default="gir"` should only be set for the native build. This way, it's clear that the cross builds are lacking introspection support.

Reply:
I don't think we need to generate them, since they aren't used by GIMP or GEGL, but we do need to be explicit about the differences between the packages.

Reply:
@ericonr I'll ponder on your suggestion for a while, but I may suddently find myself in a simpleton spoonfeed posture if it goes all haywire.. but we'll find out eventually

Reply:
It's basically adding a layer of indirection to the `vmove` conditional.

```
if [ -z "$CROSS_BUILD" ]; then
build_options_default="gir"
fi

if [ "$build_option_gir" ]; then
vmove ...
fi
```